packages/react/src/components/TextField/index.tsx

```diff
@@ -102,6 +111,17 @@ export default class TextField extends React.Component<
             </span>
           )}
         </label>
+        {description && (
+          <div className="Field__description" id={this.descriptionId}>
+            {description}
+          </div>
+        )}
+        {error && (
+          <div className="Error Field__error" id={this.errorId}>
```

Comment:
We should just be able to maintain the `Field__error` class here:

```suggestion
          <div className="Field__error" id={this.errorId}>
```